lib/t/news.ex

```diff
@@ -136,108 +136,331 @@ defmodule T.News do
+                      "question" => "worldview",
+                      "rotation" => 0
+                    },
+                    %{
+                      "zoom" => 1,
+                      "value" => "оптимистическое",
+                      "answer" => "оптимистическое",
+                      "position" => [76, 537.6666666666667],
+                      "question" => "worldview",
+                      "rotation" => 0
+                    },
+                    %{
+                      "zoom" => 1,
+                      "value" => "• добавили новую \nкатегорию стикеров — \nмировоззрение 🌎",
+                      "position" => [68.16666666666669, 221.33331807454428],
+                      "rotation" => 0,
+                      "alignment" => 1,
+                      "text_color" => "#111010",
+                      "corner_radius" => 1,
+                      "background_fill" => "#FFFFFF"
+                    }
+                  ],
+                  "background" => %{"color" => "#5E50FC"}
+                },
+                %{
+                  "size" => [390, 844],
+                  "labels" => [
+                    %{
+                      "zoom" => 0.8227475861552165,
+                      "value" => "•новый редактор текста ✍️",
+                      "position" => [23.262899318002084, 141.54110603171702],
+                      "rotation" => 0,
+                      "alignment" => 1,
+                      "text_color" => "#111010",
+                      "corner_radius" => 1,
+                      "background_fill" => "#FFFFFF"
+                    },
+                    %{
+                      "zoom" => 0.8160309784078537,
+                      "value" => "• сетка в редакторе 👇",
+                      "position" => [150.2414146415445, 210.82327200741543],
+                      "rotation" => 0,
+                      "alignment" => 1,
+                      "text_color" => "#111010",
+                      "corner_radius" => 1,
+                      "background_fill" => "#FFFFFF"
+                    },
+                    %{
+                      "zoom" => 1,
+                      "action" => "edit_story",
+                      "value" => "ПОПРОБОВАТЬ",
+                      "position" => [106.33333333333331, 697.5],
+                      "rotation" => 0,
+                      "alignment" => 1,
+                      "text_color" => "#111010",
+                      "corner_radius" => 1,
+                      "background_fill" => "#FFFFFF"
+                    }
+                  ],
+                  "background" => %{
+                    "zoom" => 1.1458749645399164,
+                    "color" => "#2A261E",
+                    "proxy" =>
+                      "https://d3r9yicn85nax9.cloudfront.net/_5KL6jlMLg75WxlTIp5PmUrXVXbrXNrPgpFao6dyjvQ/fit/1200/0/sm/0/aHR0cHM6Ly9zaW5jZS13aGVuLWFyZS15b3UtaGFwcHkuczMuYW1hem9uYXdzLmNvbS9iMzE4Yzk1My02ZmMyLTRkMjEtYWViYS01M2IxYzY0ZmY5MzE",
```

Comment:
[`s3_key = "b318c953-6fc2-4d21-aeba-53b1c64ff931"`](https://since-when-are-you-happy.s3.amazonaws.com/b318c953-6fc2-4d21-aeba-53b1c64ff931)